CRM/Dedupe/Finder.php

```diff
@@ -122,15 +122,15 @@ public static function dupesByParams(
     }
     $rgBao->params = $params;
     $rgBao->fillTable();
-    $dao = new CRM_Core_DAO();
-    $dao->query($rgBao->thresholdQuery($checkPermission));
+
+    $dao = CRM_Core_DAO::executeQuery($rgBao->thresholdQuery($checkPermission));
     $dupes = [];
     while ($dao->fetch()) {
       if (isset($dao->id) && $dao->id) {
         $dupes[] = $dao->id;
       }
     }
-    $dao->query($rgBao->tableDropQuery());
+    CRM_Core_DAO::executeQuery(($rgBao->tableDropQuery()));
```

Comment:
why is there an extra set of `(` and `)` here?

Reply:
@seamuslee001 just cos I'm generous like that :-)

Fixed now